frontend/src/app/Routes.js

```diff
@@ -16,6 +17,10 @@ function Routes() {
   return (
     <Switch>
       <Route path="/stores/new" component={NewStore} />
+      <Route path="/customer">
+        Welcome to customer screen
+      </Route>
```

Comment:
What's this?

Reply:
An expected route for the customer address